### DISCUSSION TITLE
Production readiness of Combobox with Tag

### DISCUSSION THREAD

**Discussion by jaknas** [ariakit/ariakit#3706] [General]
Hey Diego 👋 
I wanted to ask about state of Tag component (in conjunction with Combobox especially). I know that this is marked as experimental right now, with [the code](https://github.com/ariakit/ariakit/pull/3579) being there and possible to use.

I'm curious if there are some known issues with it, or what it would take for this to be "production ready" with the example added to the Ariakit website.

Many thanks for all the hard work you're doing!

**Comment by diegohaz** [comment]
Hey! I think there isn't enough time to say it's production-ready yet. But I don't have plans to make any significant changes to the component API for now.

Some folks began using it in their projects, and to this date, no problems have been reported.

Here are the unlisted examples:

- https://ariakit.org/examples/tag
- https://ariakit.org/examples/tag-combobox

Feedback is welcome.

**Reply by arnoldsandoval** [reply]
This is great @diegohaz, thanks! Im new to following this project, is there a way to find out about these experimental components in the docs? I tried searching for it using the global search but it didnt produce any results.
